HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -1037,18 +1038,19 @@ code::
 ::
 
 method:: exprand
-Returns a link::Classes/Float:: in the range [this, arg) drawn from a log-uniform distribution.
+Returns a link::Classes/Float:: in the range (this, aNumber) drawn from a log-uniform distribution.
 Receiver (i.e., code::this::) must not be zero.
 Note that the value is emphasis::not:: drawn from an exponential distribution.
-argument::arg
+argument::aNumber
 The other (upper or lower) limit of the range. Must be of same sign as code::this::, and must not be zero.
+argument::adverb
 returns:: Always returns a link::Classes/Float::.
 
 method:: gauss
-Returns a link::Classes/Float:: sampled from a Gaussian distribution with mean code::this:: and standard deviation code::stddev::.
+Returns a link::Classes/Float:: sampled from a Gaussian distribution with mean code::this:: and standard deviation code::standardDeviation::.
 The range of the gaussian distribution is unbounded and thus may return +/-inf;
 consider using link::Classes/SimpleNumber#-clip#clip::.
-argument::stddev
+argument::standardDeviation
```

Comment:
Thank you for aligning this argument name in the the helpfile with the method definition!